ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -229,6 +229,15 @@ class GraphsContainer extends React.Component {
     return moment.utc(tick).format('MMM DD hh:mm');
   };
 
+  computeYAxisLabel = () => {
+    const { measurementUnits } = this.props;
+
+    if (measurementUnits && measurementUnits.size === 1) {
+      return measurementUnits.values().next().value;
```

Comment:
Is there a more straightforward way to access the `value` field? Just asking.

Reply:
@octavian-negru how about
```javascript
return [...measurementUnits][0];
```

Reply:
Nice :)